src/commands/runtime/action/create.js

```diff
@@ -172,7 +172,7 @@ class ActionCreate extends RuntimeBaseCommand {
         limits = limits || {}
         limits.timeout = flags.timeout
       }
-      if (flags.logsize) {
```

Comment:
Should we check for `null` too? I can't really find in code where `flags.logsize` is being initialized.

Reply:
No, it's not necessary is when args are parsed into `flags` everything which is not set will be `undefined`.